web/src/html.js

```diff
@@ -38,15 +39,35 @@ function buildOgMetadata(overrideOptions = {}) {
   const head =
     '<title>lbry.tv</title>\n' +
     `<meta property="og:url" content="${URL}" />\n` +
-    `<meta property="og:title" content="${title || 'lbry.tv'}" />\n` +
-    '<meta property="og:site_name" content="lbry.tv | Content Freedom"/>\n' +
-    `<meta property="og:description" content="${description ||
-      'Meet LBRY, an open, free, and community-controlled content wonderland.'}" />\n` +
-    `<meta property="og:image" content="${URL}/v2-og.png" />\n` +
+    `<meta property="og:title" content="${title || SITE_TITLE}" />\n` +
+    `<meta property="og:site_name" content="${SITE_TITLE} | ${SITE_MOTTO}"/>\n` +
+    `<meta property="og:description" content="${description || SITE_DESCRIPTION}" />\n` +
+    `<meta property="og:image" content="${URL}/public/v2-og.png" />\n` +
     '<meta name="twitter:card" content="summary_large_image"/>\n' +
-    `<meta name="twitter:image" content="${URL}/v2-og.png"/>\n` +
+    `<meta name="twitter:title" content="${title || SITE_TITLE}" />\n` +
+    `<meta name="twitter:description" content="${description || SITE_DESCRIPTION}" />\n` +
+    `<meta name="twitter:image" content="${URL}/public/v2-og.png"/>\n` +
+    `<meta name="twitter:url" content="${URL}" />\n` +
     '<meta property="fb:app_id" content="1673146449633983" />';
+  return head;
+}
 
+function buildBasicOgMetadata() {
+  const head =
+    '    <!-- VARIABLE_HEAD_BEGIN -->' +
```

Comment:
I think this should be removed from the HTML file (but keep the comment?) now that this exists?
https://github.com/lbryio/lbry-desktop/blob/master/static/index-web.html#L66-L74